### ISSUE TITLE
resolve duplicates in handbook

### ISSUE THREAD

**Issue by tangjeff0** [athensresearch/handbook#29]
Maybe Gitbook creates some duplicates and gets out of sync, but noticing duplicates in the source files for this repo.

One example. Two directories with similar functions. `onboarding-for-new-clojurians.md` shows up 4 times.

```
school-of-athens
├── clojurefam
│   ├── clojurefam-cohort-and-team-rosters.md
│   ├── learner-commits.md
│   ├── onboarding-for-new-clojurians.md    # 1
│   └── README.md
├── clojurefam.md
├── github-guide.md
├── school-of-athens-faq.md
└── school-of-athens.md
```

```
learning-at-athens-academy-education
├── clojure
│   ├── clojurefam
│   │   ├── clojurefam-cohort-and-team-rosters.md
│   │   ├── learner-commits.md
│   │   └── README.md
│   ├── onboarding-for-new-clojurians.md    # 2
│   └── README.md
├── clojurefam
│   ├── clojurefam-cohort-and-team-rosters.md
│   ├── learner-commits.md
│   ├── onboarding-for-new-clojurians.md    # 3
│   └── README.md
├── knowledge-management.md
├── notetaking.md
├── onboarding-for-new-clojurians.md         # 4
├── README.md
└── research.md
```

**Comment by joelhans** [comment]
I spent some time poking around GitBook and GitHub last night and figured out that GitBook has a tendency to simply duplicate files when we move content from one section or nested page to another, and not delete the previous instance.

For example, if we changed the **Main Stuff** section, which stores most of the handbook content under the `/main-stuff/` folder right now, there's a good chance that GitBook will simply create a new folder and duplicate the content there.

I think that by end of day on Wednesday we should put a "code freeze" on handbook structure. That gives us at least a day to perform the cleanup required, here on GitHub, to make sure that the files here are 100% representative of what we're publishing on GitHub.
